<?xml version="1.0"?>
<launch>

  <node pkg="flightcontroller" exec="velocity_controller.py" name="velocity_controller_node" output="screen">
    <param name="/velocity_controller_node/gains/p_xy" type="float" value="0.3" />
    <param name="/velocity_controller_node/gains/i_xy" type="float" value="0.003" />
    <param name="/velocity_controller_node/gains/d_xy" type="float" value="0.01" />
    <param name="/velocity_controller_node/gains/p_z" type="float" value="0.75" />
    <param name="/velocity_controller_node/gains/i_z" type="float" value="0.06" />
    <param name="/velocity_controller_node/gains/d_z" type="float" value="0.06" />
  </node>

</launch>